lib/cocoapods/installer/project_cache/project_cache_analyzer.rb

```diff
@@ -78,7 +84,8 @@ def analyze
           # Bail out early since these properties affect all targets and their associate projects.
           if cache.build_configurations != build_configurations ||
               cache.project_object_version != project_object_version ||
-              YAMLHelper.convert(cache.podfile_plugins) != YAMLHelper.convert(podfile_plugins)
+              YAMLHelper.convert(cache.podfile_plugins) != YAMLHelper.convert(podfile_plugins) ||
+              YAMLHelper.convert(cache.installation_options) != YAMLHelper.convert(installation_options)
```

Comment:
I forget if this is necessary. I thought ruby hash equality didn't care about key orderings, but it might have with the podfile plugins?

Reply:
maybe it had to do with hash with indifferent access vs normal hash for plugins, in which case i think the concern might still apply? not 100% sure